ts/Core/Series/Series.ts

```diff
@@ -7269,6 +7269,30 @@ class Series {
         fireEvent(series, selected ? 'select' : 'unselect');
     }
 
+    /**
+     * Checks if a tooltip should be shown for a given point.
+     *
+     * @private
+     * @param {number|true} plotX
+     * @param {number|true} plotY
+     * @return {boolean}
+     */
+    public shouldShowTooltip(
```

Comment:
Instead of `{number|true}`, maybe it would be cleaner here too to follow the pattern from `Chart.isInsidePlot` and have pure numeric `plotX` and `plotY`, then `options` as the third argument? Then the third argument can just be extended with `series` and possibly `visiblePlotOnly`, and everything passed on to `Chart.isInsidePlot`.